cmd/go-quai/start.go

```diff
@@ -143,3 +145,11 @@ func EnablePprof() {
 		log.Global.Print(http.ListenAndServe("localhost:"+port, nil))
 	}()
 }
+
+func EnableLibp2pMetrics() {
+	http.Handle("/metrics", promhttp.Handler())
+	go func() {
+		http.Handle("/debug/metrics/prometheus", promhttp.Handler())
+		log.Global.Fatal(http.ListenAndServe(":5001", nil))
```

Comment:
What happens if "http.ListenAndServe(":5001", nil)" returns (due to an error or otherwise)? Does the whole node crash or just this goroutine?

Reply:
the whole node, as intended